WalletWasabi/BlockchainAnalysis/Cluster.cs

```diff
@@ -13,10 +13,14 @@ public class Cluster : NotifyPropertyChangedBase
 		private string _labels;
 
 		public Cluster(SmartCoin coin)
+			: this( new[] { coin })
 		{
-			Labels = "";
-			Coins = new List<SmartCoin>() { coin };
-			Labels = string.Join(", ", coin.Label.Labels);
+		}
+
+		public Cluster(IEnumerable<SmartCoin> coins)
+		{
+			Coins = coins.ToList();
+			Labels = string.Join(", ", KnownBy);
```

Comment:
ctor1: public SmartLabel(params string[] labels) : this(labels as IEnumerable<string>)
ctor2: public SmartLabel(IEnumerable<string> labels)

Reply:
Yes, good pattern. Done.